apollo-federation/tests/query_plan/build_query_plan_tests/interface_object.rs

```diff
@@ -797,33 +789,33 @@ fn it_handles_interface_object_input_rewrites_when_cloning_dependency_graph() {
               }
             },
             Parallel {
-              Flatten(path: "i") {
-                Fetch(service: "S2") {
+              Flatten(path: "i.i2") {
+                Fetch(service: "S4") {
```

Comment:
We are okay that this snapshot changed and we are picking a different service than before? I am asking because I am fairly certain I've seen quite a few mismatches that resembled this sort of change.